statistics/handle/handle.go

```diff
@@ -354,7 +355,11 @@ func (h *Handle) RemoveLockedTables(tids []int64, pids []int64, tables []*ast.Ta
 }
 
 // IsTableLocked check whether table is locked in handle
-func (h *Handle) IsTableLocked(tableID int64) bool {
+func (h *Handle) IsTableLocked(tableID int64, isLock bool) bool {
+	if isLock {
+		h.mu.RLock()
+		defer h.mu.RUnlock()
+	}
```

Comment:
`isLock bool` is a bad behavior here. If we want use `IsTableLocked` with locked, we should use public method `IsTableLocked`, otherwise we should use private method `isTableLocked`

Reply:
add ```isTableLocked```.